clouddriver-ecs/clouddriver-ecs.gradle

```diff
@@ -1,3 +1,17 @@
+sourceSets {
+  integration {
+    java.srcDirs = ["src/integration/java"]
+    resources.srcDirs = ["src/integration/resources"]
+    compileClasspath += main.output + test.output
```

Comment:
Out of curiosity, what happens if you don't include main.output+test.output on the compileClasspath?

Reply:
so w/o the "compileClasspath" line the integ tests won't build, but I tried it w/o the "runtimeClasspath" and it seems to build/run fine! will remove that one (iirc I originally added it based on reference docs but the specific usage must have been different).